modules/base/main.tf

```diff
@@ -0,0 +1,181 @@
+resource "azurerm_resource_group" "rg" {
```

Comment:
Next step: fill these parameters for the quickstart template!\n after fill, resolve the comment [and](https://github.com/Azure/Edge-infrastructure-quickstart-template/blob/main/doc/Scale-with-automation.md#methodology)